tests/scripts/run-xunit-perf.cmd

```diff
@@ -78,6 +78,11 @@ xcopy /s %BENCHDIR%*.txt . >> %RUNLOG%
 
 set CORE_ROOT=%CORECLR_REPO%\sandbox
 
+@rem setup additional environment variables
+if EXIST %TEST_ENV% (
```

Comment:
I know we are generally super lazy about this, but the lack of quotes here makes me nervous.  My understanding is that the preferred pattern is to refer to command line arguments by %~2 instead of %2, which will remove enclosing quotes if they exist.  Then you should be free to add quotes from from that point forward.  Not a big deal in this case, but feel free to change if you like.